app_tests/integration_tests/llm/sglang/sglang_frontend_test.py

```diff
@@ -0,0 +1,309 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import re
+import pytest
+
+from ..utils import (
+    AccuracyValidationException,
+)
+
+pytest.importorskip("sglang")
+import sglang as sgl
+from sglang.lang.chat_template import get_chat_template
+
+pytest.importorskip("sentence_transformers")
+from sentence_transformers import SentenceTransformer, util
+
+logger = logging.getLogger(__name__)
+
+DEVICE_SETTINGS = {
+    "device_flags": [
+        "--iree-hal-target-backends=rocm",
+        "--iree-hip-target=gfx942",
+    ],
+    "device": "hip",
+}
+
+ACCEPTED_THRESHOLD = 0.8
+
+
+def compute_similarity(model: SentenceTransformer, sentence_1: str, sentence_2: str):
```

Comment:
NICE!

If we end up needing a faster version of this CI though, we might need a smaller model / another model free approach to compute sentence similarity.

Reply:
Agreed, most of the other solutions required way too much setup to be able to get embeddings. But, there should be a way to do this without loading a model at all. 

I considered seeing if I could get the embedding with shortfin LLM itself, but that felt like using shortfin tokens to validate shortfin tokens